webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/EditorTechniqueReader.scala

```diff
@@ -67,29 +67,26 @@ class TechniqueReader(
       result
     }
   }
-  def readTechniquesMetadataFile:              IOResult[(List[EditorTechnique], Map[BundleName, GenericMethod], List[RudderError])] = {
-    import yamlTechniqueSerializer._
 
-    import zio.json.yaml._
```

Comment:
Here inmport should have been our yamlOps so zio.yaml, this would have allow to keep using fromYaml (but even i'm not sure there is a bug in decoding json, i think it just on encoding side)

Reply:
ok understood why you not use it anymore